torch/csrc/jit/fuser/cuda/fused_kernel.cpp

```diff
@@ -108,13 +108,21 @@ FusedKernelCUDA::FusedKernelCUDA(
   AT_CUDA_NVRTC_CHECK(nvrtc().nvrtcCreateProgram(
       &program, code_.c_str(), nullptr, 0, nullptr, nullptr));
 
+#ifdef __HIP_PLATFORM_HCC__
+  std::vector<const char*> args = {};
+#else
   const std::string compute = "--gpu-architecture=compute_" +
       std::to_string(major) + std::to_string(minor);
   const std::vector<const char*> args = {
       "--std=c++11", compute.c_str(), "-default-device"};
+#endif
   const auto result =
       nvrtc().nvrtcCompileProgram(program, args.size(), args.data());
+#ifdef __HIP_PLATFORM_HCC__
+  if (result != HIPRTC_SUCCESS) {
```

Comment:
Could this be hipified?

Reply:
Only if I can switch the CUDA path to also do the catch-all for `!= NVRTC_SUCCESS`.

Reply:
Yes I think changing to do catch-all is reasonable.